app/admin/quotes.rb

```diff
@@ -0,0 +1,9 @@
+ActiveAdmin.register Quote do
+
+  permit_params :description, :author_name, :order
+
+  before_create do |quote|
+    quote.id = quote.order
```

Comment:
@SergeyProger If you are trying to set order to the next available one, here is a way you can do that:
```ruby
quote.order = Quote.maximum(:order)+1
``` 
In general you don't want to try to override primary key values under any circumstances, only trouble can come from that.